static-site/src/components/ListResources/Showcase.tsx

```diff
@@ -97,6 +101,27 @@ const ShowcaseTile = ({card, setSelectedFilterOptions}: ShowcaseTileProps) => {
 }
 
 
+interface UrlShowcaseTileProps {
+    card: UrlCard
+}
+
+
+const UrlShowcaseTile = ({ card }: UrlShowcaseTileProps) => {
```

Comment:
> I decided not to implement abstract on hover (from https://github.com/nextstrain/nextstrain.org/issues/849#issue-2276702417) here. I plan to explore that in a separate PR later. Should it be included in this PR?

I don't think it has to be added in this PR, and we can still release this design without them, but I do think surfacing a small abstract/description for each card¹ is a critical part of where we want to go. So if it's not done now we should prioritise doing it next.

¹ It doesn't have to use my hover design, but the proposal's been up for over a month and there haven't been any competing designs shared.

Reply:
Mentioned this in https://github.com/nextstrain/nextstrain.org/pull/901#discussion_r1633900079